src/components/Composer/index.js

```diff
@@ -407,7 +339,19 @@ class Composer extends React.Component {
 Composer.propTypes = propTypes;
 Composer.defaultProps = defaultProps;
 
+const ComposerWithDragAndDrop = props => (
```

Comment:
Good job extracting a common component out of `DragAndDrop`, however our code style across this repo is generally one-component-per-file. You should:

1. Include the `DragAndDrop` component in the main `Composer` component.
1. Spread the `DragAndDrop` prop types in the main prop types for the `Composer` component.
1. Get rid of the `ComposerWithDragAndDrop` component.